enterprise/signals.py

```diff
@@ -165,7 +165,7 @@ def assign_or_delete_enterprise_admin_role(sender, instance, **kwargs):     # py
                 enterprise_customer=instance.enterprise_customer,
             )
             # Also create the Enterprise admin user in third party analytics application
-            create_tableau_user(instance.user.id, instance)
+            create_tableau_user(str(instance.enterprise_customer.uuid), instance)
```

Comment:
Add a comment here explaining that we are creating a user with username set to the uuid of the enterprise